bot/moderation/moderation.py

```diff
@@ -140,11 +188,11 @@ async def modmail_reaction_add(self, payload: discord.RawReactionActionEvent):
         Args:
             payload (discord.RawReactionActionEvent): The payload for the triggered event.
         """
+        # TODO: Check if user has Moderator role.
```

Comment:
We will have to do this for many commands. 
We should create an issue for this that is part of v1.0 so we won't forget it.